charts/drax/values.yaml

```diff
@@ -245,7 +245,6 @@ du-metrics-server:
       port: 80
       targetPort: 8086
 
-
 e2-t:
```

Comment:
Undo the deletion of this blank line.

The double blank line is added with the intention to separate charts from each other (within chart values you can then use a single blank line).

Reply:
OK